scripts/results/session-nack.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+#.....................................................................
+# Copyright (c) 2016-2020, Regents of the University of Arizona.
+# Author: Chavoosh Ghasemi <chghasemi@cs.arizona.edu>
```

Comment:
No, you should include yourself AND all the sources you used, as well.
This should be like this:
```
Authors: Wenkai Zhang <....>
         Chavoosh ....
```